README.md

```diff
@@ -8,21 +8,55 @@ Welcome to the editing environment for the DE-TEL Book.
 
 ## Prepare Development Environment
 
-1. Install [Ruby](https://www.ruby-lang.org) on your machine ([Detailed instructions](https://jekyllrb.com/docs/installation/))
+__Step 1.__ Install [Ruby](https://www.ruby-lang.org) on your machine ([Detailed instructions](https://jekyllrb.com/docs/installation/))
 
-2. Clone this repository and open a terminal / command line
+If you are going to install Ruby on a Windows machine, there is a convenient tool [RubyInstaller](https://rubyinstaller.org/downloads/).
 
-   Check that Ruby is successfully installed and can be accessed from the terminal / command line.
-   To do this, run the command "ruby -v".
-   It should return the installed version of Ruby.
+Ruby version 2.7.5 is recommended to work with this repository.
 
-3. In the terminal/command line, navigate to your main project folder
+__Step 2.__ Check that Ruby is successfully installed and can be accessed from the terminal / command line.
+To do this, open the Command prompt and run command:
+   
+    ruby -v
+   
+It should return the installed version of Ruby.
 
-4. Install bundler
+__Step 3.__ Clone this DETEL book repository. If needed, use instructions of [how to clone a github repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository). Remember the folder where the local version of the repository is stored on your machine.
+
+
+__Step 4.__ In the terminal/command line, navigate to the folder with the local version of the DETEL book repository. To navigate to the correct folder, use following command:
+
+    cd

```

Comment:
Maybe add here that cd requires a parameter so that people who read the instructions top to bottom and immediately try out every line before reading on do not just type in the command cd.

```suggestion
    cd foldername

```